src/translations/en-US.json

```diff
@@ -300,18 +300,23 @@
       "limit": 0
    },
    "visualizations.menu.aggregations": {
-      "value": "Aggregations",
-      "comment": "",
-      "limit": 0
+      "value": "Aggregate",
+      "comment": "Heading showed above menu listing possible aggregations (totals).",
+      "limit": 50
    },
    "visualizations.menu.aggregations.rows": {
       "value": "Rows",
-      "comment": "Rows of a table",
+      "comment": "Heading showed above row attributes of the table you may apply one of aggregation (total) on.",
       "limit": 25
    },
    "visualizations.menu.aggregations.all-rows": {
-     "value": "All rows",
-     "comment": "All rows of a table",
+     "value": "of all rows",
+     "comment": "Example: Sum (total) 'of all rows' will be showed in the table",
```

Comment:
I would write something similar: "The text will be displayed as a menu option that adds row grand total to the table. It's meant to be read as sentence "Sum of all rows" where "Sum" is the option in the menu and "of all rows" is the first option in the "Sum" menu option submenu.

Reply:
Agree, but we should to figure out sth shorter than your proposal